crowdin_api/api_resources/string_comments/resource.py

```diff
@@ -74,7 +75,7 @@ def add_string_comment(
 
         return self.requester.request(
             method="post",
-            path=self.get_string_comments_path(projectId=projectId),
+            path=self.get_string_comments_path(projectId=projectId, stringCommentId=stringId),
```

Comment:
This change shouldn't be there since the `Add String Comment` API doesn't require the `stringCommentId`.

You need to pass the new `stringId` into the `request_data` object